### ISSUE TITLE
Crash on creative inventory + crafting table crash

### ISSUE THREAD

**Issue by ShadowMine64** [zhuowei/MCPELauncher#377]
I keep getting a crash when I open creative inventory and when I use the crafting table, I also crash.

**Comment by zhuowei** [comment]
which scripts?

**Comment by ShadowMine64** [comment]
I found out that the crafting table crash is with recipes, I'll try to send the script

**Comment by ingvaras** [comment]
Yeah. Same bug for me. Crafting recipes worked with my mod in 0.12.3 and older bl. But doesnt work now. This must be a bug. But if you want to see code:

ModPE.setItem(714, "healeritem", 0, "Healer");
Item.addShapedRecipe(714, 1, 0, [
"iii",
"idi",
"iii"
],
["i", 265, 0, "d", 264,0]);

**Comment by zhuowei** [comment]
Does this still happen in the latest beta?

**Comment by ShadowMine64** [comment]
Nope everything works now :)
